malaria24/settings/base.py

```diff
@@ -117,6 +118,11 @@
 
 DEFAULT_FROM_EMAIL = 'MalariaConnect <malaria24@praekelt.com>'
 
+# JEMBI settings
+URL = 'http://jembi.org/malaria24'
```

Comment:
I would just make this more specific, maybe prefix these with `JEMBI_` so that it's easy to see what it is for.

Reply:
Okay, will add that, thanks